doc/source/serve/index.rst

```diff
@@ -44,23 +44,133 @@ Ray Serve can be used in two primary ways to deploy your models at scale:
 Ray Serve Quickstart
 ====================
 
-Ray Serve supports Python versions 3.6 through 3.8. To install Ray Serve, run the following command:
+Ray Serve supports Python versions 3.6 through 3.9.
```

Comment:
maybe just link to Ray's support matrix: https://docs.ray.io/en/master/installation.html